docs/source/usage.rst

```diff
@@ -96,14 +96,26 @@ Changing processors
 
 .. automethod:: kordac.Kordac.processor_defaults(processors)
 
-  This function is useful if you want to make minor changes to the default used processors. For example: with an existing Kordac converter ``converter``, you wish to still use all default processors but now skip video tags:
+  This function is useful if you want to make minor changes to the default used processors. For example: You wish to still use all default processors but skip video tags:
 
   .. code-block:: python
 
-    processors = converter.processor_defaults()
+    processors = Kordac.processor_defaults()
+    processors.remove('video')
+    converter = Kordac(processors=processors)
+
+  .. _changing processors :
```

Comment:
Can you tell me what this line is for (also line 117)? I don't think it's even valid Sphinx notation.